design/workitems.go

```diff
@@ -288,3 +288,26 @@ var _ = a.Resource("redirect_workitem", func() {
 		a.Response(d.MovedPermanently)
 	})
 })
+
+var _ = a.Resource("planner_backlog", func() {
+	a.Parent("space")
+	a.BasePath("/backlog")
+
+	a.Action("list", func() {
+		a.Routing(
+			a.GET(""),
+		)
+		a.Description("List backlog work items.")
+		a.Params(func() {
+			a.Param("page[offset]", d.String, "Paging start position")
+			a.Param("page[limit]", d.Integer, "Paging size")
```

Comment:
We need to support some of the other filters from the normal wi list here, e.g. Assigned to, Area. I would assume all that is not iteration.. unsure if state would be a valid filter.